src/const.ts

```diff
@@ -224,3 +224,8 @@ export const DISABLED_TOKEN_MAPS = Object.keys(disabledTokens).reduce<DisabledTo
     [Network.xDAI]: {},
   },
 )
+
+export enum AnalyticsDimension {
```

Comment:
Since this is a const, shouldn't it be all caps?

Reply:
actually is a enum so i moved it to types